backend/tests/e2e/conftest.py

```
@@ -0,0 +1,471 @@
+unit or integration tests. All environment variables, database files,
+and global state are properly isolated and cleaned up.
+"""
+
+import os
+import tempfile
+import uuid
+import logging
+import sys
+from contextlib import contextmanager
+from pathlib import Path
+from typing import Generator
```

Ruff (0.12.2)

`typing.Generator` imported but unused

Remove unused import: `typing.Generator`

(F401)